draft-ietf-rtcweb-jsep.xml

```diff
@@ -841,15 +846,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               of the encoding. For example, if the encoder resolution is
               1280x720, and the attribute specified a maximum of 640x480, the
               expected output resolution would be 640x360.
-              If downscaling cannot be applied, the encoding MUST NOT be
-              transmitted, and an error SHOULD be raised to the application.</t>
+              If downscaling cannot be applied, this MUST be treated as an error.</t>
 
               <t>If the encoder resolution is less than the minimum size permitted
-              by the attribute, the encoding MUST NOT be transmitted, and an
-              error SHOULD be raised to the application; the encoder
+              by the attribute, this MUST be treated as an error; the encoder
               MUST NOT apply upscaling. JSEP implementations SHOULD avoid this
               situation by allowing receipt of arbitrarily small resolutions,
               perhaps via fallback to a software decoder.</t>
+
+              <t>If the encoder resolution is within the maximum and minimum
+              sizes, no action is needed.</t>
```

Comment:
I am generally good with this, but I think "MUST be treated as an error" is confusing. Perhaps "this set of attributes MUST NOT be used"?

Reply:
I think you mean 'this attribute MUST NOT be used', but I think this is a good suggestion; I wasn't completely happy with the current text either.

Reply:
Updated accordingly